### ISSUE TITLE
Please add a license

### ISSUE THREAD

**Issue by karan** [iDigBioWorkflows/FlatSheetsDigitizatonWorkflows#1]
This repo is missing a license. Without a license, all code is copyright the author and may not be used by anyone else.

Please use something like http://choosealicense.com/ to decide what license to use. I recommend MIT or GPL.

**Comment by iDigBioWorkflows** [comment]
Since this is not code, can I make it CC BY? If so, where to I add this?

On 9/14/2015 2:48 PM, Karan Goel wrote:

> This repo is missing a license. Without a license, all code is 
> copyright the author and may not be used by anyone else.
> 
> Please use something like http://choosealicense.com/ to decide what 
> license to use. I recommend MIT or GPL.
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/iDigBioWorkflows/FlatSheetsDigitizationWorkflows/issues/1.Web 
> Bug from 
> https://github.com/notifications/beacon/ALCP2NhT6Aru3ceECZoPCz38JnXdhGpVks5oxw35gaJpZM4F9EWb.gif

## 

Gil Nelson, PhD
Assistant Professor/Research
iDigBio Steering Committee
Integrated Digitized Biocollections
Institute for Digital Information and Scientific Communication
College of Communication and Information

Courtesy Professor
Department of Biological Sciences
Robert K. Godfrey Herbarium
Florida State University
gnelson@bio.fsu.edu

**Comment by debpaul** [comment]
Hi @iDigBioWorkflows, stack overflow says:
1. Go to your repository. (for example: https://github.com/iDigBioWorkflows/FlatSheetsDigitizationWorkflows)
2. Click on "Create new file" Button.
3. Type the file name as License.txt or License.md in the input box next to your repository name, a drop down button appears towards right side.
4. Choose the type of license of your choice.
5. Click "Commit new file" button at the bottom (Green button)

Let me know if you want me to do this. :-)

**Comment by debpaul** [comment]
Yes @iDigBioWorkflows, I do see other GitHub repos using cc licensing.
